### ISSUE TITLE
What Replaced 'HTTP_TEXT'?

### ISSUE THREAD

**Issue by ghost** [tzapu/WiFiManager#800]
#### Hardware
**WiFimanager Branch/Release:**
- [X] Master
- [ ] Development

**Esp8266/Esp32:**
- [X] ESP8266
- [ ] ESP32

**Hardware:			ESP-12e, esp01, esp25**
- [ ] ESP01
- [X] ESP12 E/F/S (nodemcu, wemos, feather)
- [ ] Other

**ESP Core Version:	2.4.0, staging**
- [ ] 2.3.0
- [X] 2.4.0
- [ ] staging (master/dev)

### Description

One used to be able to edit HTTP_TEXT[] PROGMEM inside WiFiManager.h to add text, now with latest version it simply says "WiFiManager" and HTTP_TEXT[] is gone. What has replaced it, as a way to add customized text to the landing page BEFORE the calling /wifi/ form action?

### Settings in IDE

Module:  			Wemos D1

Additional libraries: N/A

**Comment by tablatronix** [comment]
no idea, development version might be easier to modify

**Comment by h4waii** [comment]
Development branch moved the entries to strings.h but there is still no HTTP_TEXT[] entry. 

It would appear that it comes from WiFiManager.cpp lines 423 to 434, but changing this and rebuilding does not reflect any changes.

**Comment by tablatronix** [comment]
you can also set setCustomHeadElement() but it will not replace just append

**Comment by h4waii** [comment]
That would be great if it could replace instead of appending.

I'd also request removing the `_apName` and `<h3>` hardcoded text and simply replacing it with an entry from PROGMEM. I did like `HTTP_TEXT[]` as it was named well and obvious what it did.

```
String page = FPSTR(HTTP_HEAD);
page.replace("{v}", "Options");
page += FPSTR(HTTP_SCRIPT);
page += FPSTR(HTTP_STYLE);
page += _customHeadElement;
page += FPSTR(HTTP_HEAD_END);
page += String(F("<h1>"));
page += _apName;
page += String(F("</h1>"));
page += String(F("<h3>WiFiManager</h3>"));
page += FPSTR(HTTP_PORTAL_OPTIONS);
page += FPSTR(HTTP_END);
```

**Comment by tablatronix** [comment]
were you using kentaylors branch maybe ?

**Comment by h4waii** [comment]
Initially I was using an old branch that still had HTTP_TEXT, but went an did a full board (to 2.4.2) and library (to 0.14) update and my customized changes didn't apply. Was always with official releases though.

**Comment by tablatronix** [comment]
I don't think so, must have been a PR or something, no way text would have been moved from .h to .cpp and I see no history of HTTP_TEXT

**Comment by h4waii** [comment]
You're right, I searched through and never saw HTTP_TEXT either. Not sure, but anyway, if we can make these changes, then it will be easier for users to customize the pages.

**Comment by tablatronix** [comment]
well that header is only on root

**Comment by h4waii** [comment]
No, it's fine to simply leave it on root.

I believe `HTTP_HEAD` and `HEAD_END` is on every page and can be used to "globally" customize every page.

**Comment by h4waii** [comment]
https://github.com/tzapu/WiFiManager/pull/835
